moped-editor/src/views/projects/projectView/ProjectMilestones/RelatedPhaseTextField.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { TextField } from "@mui/material";
+import { makeStyles } from "@mui/styles";
+import { useGridApiContext } from "@mui/x-data-grid-pro";
+
+const useStyles = makeStyles((theme) => ({
+  readOnlyInput: {
+    width: "90%",
+    paddingTop: "inherit",
+    paddingLeft: "9px",
```

Comment:
What do you think about updating this padding left to `theme.spacing(1)`? Is this padding to keep it from resting right against the blue outline when focused?

Reply:
now I think im seeing things, because I could have sworn that last night 9px was the padding it needed to line up with the field above it. ill change it to theme spacing